ThunderCloud/QuizBadgeCollectionCell.swift

```diff
@@ -182,8 +182,10 @@ open class QuizBadgeCollectionCell: CollectionCell {
             
         } else {
             
-            quizQuestionViewController.hidesBottomBarWhenPushed = true
-            parentViewController?.navigationController?.pushViewController(quizQuestionViewController, animated: true)
+            let navigationController = parentViewController?.navigationController
```

Comment:
@simonmitchell This is new from the previous PR btw

Reply:
Nice catch!